spotify_player/src/config/keymap.rs

```diff
@@ -397,6 +397,7 @@ impl KeymapConfig {
         self.keymaps
             .iter()
             .find(|&keymap| keymap.key_sequence == *key_sequence)
+            .filter(|keymap| keymap.command != Command::None)
```

Comment:
you can include this condition in `find`, e.g `keymap.key_sequence == *key_sequence && keymap.command != Command::None`

Reply:
It was a stylistic preference :D, but if you prefer the other way, it's also fine.